db/mysql/ops.go

```diff
@@ -47,7 +47,7 @@ func (c *client) SaveMessage(ctx context.Context, id string, message *domain.Mes
 }
 
 func (c *client) ListChatRoom(ctx context.Context) ([]*domain.ChatRoom, error) {
-	rows, err := c.dbc.Query("SELECT id, name FROM chatrooms")
+	rows, err := c.dbc.Query("SELECT c.id, c.name, ru.user_id FROM chatrooms c JOIN room_user ru ON c.id = ru.room_id ")
```

Comment:
In the chatrooms table, I couldn't add an array of users because there is no such option in sql for that. room_user is a table in chat_app.sql. 

Yes, it will return chatrooms and its associated users. Also, you are absolutely right. This happened because of my poor design choice in the model layer in the first place. Had to tweak now to make all of this work.